as/src/fabric/clustering.c

```diff
@@ -1348,7 +1348,8 @@ if (((size) > STACK_ALLOC_LIMIT()) && buffer) {cf_free(buffer);}
 #define TRACE_LOG(context, format, ...)
 #endif
 
-#define CF_TRACE CF_FAULT_SEVERITY_UNDEF
+//bri//#define CF_TRACE CF_FAULT_SEVERITY_UNDEF
```

Comment:
leave comments of prior code out of the commit.

Reply:
commit message should be more explicit, and its not a 'hack', its a propper fix'